### DISCUSSION TITLE
Add the clone repo or import a repo

### DISCUSSION THREAD

**Discussion by eyekay234** [community/community#10617] [Mobile]
Is there a feature where a user can clone a repo from the Android app or import a repo

**Comment by brianlovin** [comment]
Thanks for the note @eyekay234 — out of curiosity, if you could clone a repository to your phone, what would you want to do next with those files?

**Reply by eyekay234** [reply]
I'm not cloning to my phone, more like importing the repo like i could do from the web console

**Reply by brianlovin** [reply]
I see, so the code is on your phone in a file directory somewhere and you want to upload it to GitHub through the Android app?
